code/game/machinery/vending_types.dm

```diff
@@ -376,8 +376,8 @@
 	req_access = list(access_medical_equip)
 	vend_id = "meds"
 	products = list(
-		/obj/item/reagent_containers/glass/bottle/antitoxin = 4,
-		/obj/item/reagent_containers/glass/bottle/norepinephrine = 4,
+		/obj/item/reagent_containers/glass/bottle/antitoxin = 6,
+		/obj/item/reagent_containers/glass/bottle/norepinephrine = 6,
```

Comment:
This is absolutely way too much IMO. There are at least four vendors in medical, and these medications can be made in the pharmacy, or ordered by cargo. Medical is already way too well stocked.
Everything else is fine, though I'm hesitant on the bruise/burn packs.

Reply:
I disagree on the way too well stocked notion, but I'm not opposed to just keeping it at 4. I'll bump the kits down to 4 each as well.

Reply:
Nevermind. Putting that back to normal, but adding gauze and ointment to the vendors too.